cypress-shared/support/fedex-shipping/graphql_testcase.js

```diff
@@ -121,6 +141,9 @@ export function verifyDockisActive(response) {
 }
 
 export function warehouse(id) {
+  cy.qe(
```

Comment:
We also passing variables here 
cy.qe("Verifying Inventory is unlimited via graphql")
cy.qe("The graphQl query is,query($sku: ID!, $warehouseId: ID!){inventoryProduct(sku:$sku,warehouseId:$warehouseId){unlimited}}")
cy.qe(`Variables - {id: ${id}}`)

<img width="654" alt="Screen Shot 2023-02-20 at 2 17 02 PM" src="https://user-images.githubusercontent.com/92449902/220056627-3e2e7970-6050-4b8c-8239-0c583eec9f70.png">

Please do mention that